examples/subplots_axes_and_figures/multiple_figs_demo.py

```diff
@@ -10,10 +10,11 @@
 
 .. note::
 
-    We discourage working with multiple figures in pyplot because managing
-    the *current figure* is cumbersome and error-prone. Instead, we recommend
-    to use the object-oriented approach and call methods on Figure and Axes
-    instances.
+    We discourage working with multiple figures through the implicit pyplot interface
+    because managing the *current figure* is cumbersome and
```

Comment:
```suggestion
    We discourage working with multiple figures through the implicit pyplot
    interface because managing the *current figure* is cumbersome and
```